.travis.yml

```diff
@@ -66,7 +105,7 @@ before_script:
 
 # Run PHPs run-tests.php
 script:
-    - ./sapi/cli/php run-tests.php -p `pwd`/sapi/cli/php $(if [ $ENABLE_DEBUG == 0 ]; then echo "-d opcache.enable_cli=1 -d zend_extension=`pwd`/modules/opcache.so"; fi) -g "FAIL,XFAIL,BORK,WARN,LEAK,SKIP" --offline --show-diff --show-slow 1000 --set-timeout 120
+    - ./sapi/cli/php run-tests.php -P $(if [ $ENABLE_OPCACHE == 1 ]; then echo "-d opcache.enable_cli=1 -d zend_extension=`pwd`/modules/opcache.so"; fi) -g "FAIL,XFAIL,BORK,WARN,LEAK,SKIP" --offline --show-diff --show-slow 1000 --set-timeout 120
```

Comment:
On macOS, cli_set_process_title need root permission.  So, `sudo` may improve test results.

See also:
https://stackoverflow.com/questions/24678438/php-cli-set-process-title-had-an-error-not-initialized-correctly-on-mac-osx

Reply:
Good point, although I think I prefer to make the test skip if on `osx` and `non-root`. There are already some tests skipping depending on the user being root or not.